### PR TITLE
feat: update ui of map badges on list view

### PR DESCRIPTION
**Pull request by benfurber**
## PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix (fixes an issue)
- [x] Feature (adds functionality)
- [ ] Code style update
- [ ] Refactoring (no functional changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation changes
- [ ] Other... Please describe:

## What is the new behavior?

Adds the main updates to the vertical scroll. 

Screenshot 2024-12-13 at 16 45 46
Screenshot 2024-12-13 at 16 46 08

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Git Issues

Part of #4003

### CODE REVIEW

File: packages/components/src/MapFilterProfileTypeCardList/MapFilterProfileTypeCardList.tsx [hunk 1]
```diff
@@ -1,161 +1,72 @@
...
-    </Flex>
+    <VerticalList>
+      {items.map((item, index) => {
+        const active = activeFilters.find(
+          (filter) => item.label === filter.label,
+        )
+        return (
+          <CardButton
+            onKeyDown={(ev: KeyboardEvent) => {
+              ev.code === 'Enter'
```
Comment (mariojsnunes) [inline packages/components/src/MapFilterProfileTypeCardList/MapFilterProfileTypeCardList.tsx:38]: is some logic missing here?

**Comment by davehakkens** [comment]
preview link only loads members? 
And if I test locally with live database I see the others, but cant test on mobile ..
So for me also happy to merge an iterate on the go

image

File: packages/components/src/MapFilterProfileTypeCardList/MapFilterProfileTypeCardList.tsx (hunk 1 above)
Comment (benfurber) [reply packages/components/src/MapFilterProfileTypeCardList/MapFilterProfileTypeCardList.tsx]: I'm not really sure it's importance so removing.

**Comment by benfurber** [comment]
Shouldn't be visible for one, will fix before merging.

I appreciate the data quality stuff is annoying for QA. Data will be much better as we move more docs to the new DB service as we'll setup probably seed scripts.

**Pull request merged**